### DISCUSSION TITLE
Postgres layer missing_dataprovider

### DISCUSSION THREAD

**Discussion by GonLes** [opengisch/QField#4419] [Q&A]
hello! please help, I have layers in postgres, which is in an azure database service (I already gave permissions to the qfield ip in the azure firewall). These layers I include them to upload them to qfieldcloud however when I check the app on my mobile the layer appears with an error message indicating that there is a problem or is misconfigured.

this is the info of the qfieldcloud jobs 

QGIS Layers Data
04:59:37.096 QGSMSGLOG WARNING  1 unavailable layer(s) found:
04:59:37.096 QGSMSGLOG WARNING   * dbname='xxxx' host=xxxx port=5432 sslmode=allow authcfg=67je821 key='uuid' type=Point checkPrimaryKeyUnicity='1' table="public"."gen_point" (geom)
04:59:37.102 root      INFO     QGIS project layer checks
Layer Name                      Layer Id    Is Valid    Status                Error Summary    Provider Summary
OpenStreetMap               ...1a391a   True        no_error
                                         ...4084a0   False       missing_dataprovider                   No data provider available

** for privacy reasons use xxx in dbname and host

- I have already configured the secrets in qfieldcloud
- In the qgis project file is checked the box to save user and password of the database . 

I have reviewed the documentation posted on the site about it but it does not fix it.

thanks in advance.

Gonzalo.

**Comment by sigsdis30** [comment]
Hi, 
Is there any progress on this matter ?
Thanks
